.azure-pipelines/azure-pipelines-linux-clang.yml

```diff
@@ -14,35 +14,41 @@ jobs:
           arm_arch_target: 'armv8-a'
         clang_6:
           llvm_version: '6.0'
-        clang_7:
```

Comment:
Why disabling this one and the one below?

Reply:
There was no specific reason other than that I meant to be "careful with resources". The total number of build jobs remains the same, and I removed only intermediate versions, in the hope to still cover a broad range of clang versions. If you do not mind the extra CPU hours I can add the jobs back?

Reply:
But personally I would rather add more diverse compilers like an Android build or other diverse platforms...

Reply:
+1, just wanted to know the motivation. Thanks for the cleanup!

Reply:
Great!